redash/query_runner/__init__.py

```diff
@@ -67,6 +67,24 @@ def configuration_schema(cls):
     def run_query(self, query):
         raise NotImplementedError()
 
+    def fetch_columns(self, columns):
+        column_names = []
+        duplicates_counter = 1
+        new_columns = []
+
+        for col in columns:
+            column_name = col[0]
+            if column_name in column_names:
+                column_name = "{}_{}".format(column_name, duplicates_counter)
```

Comment:
I've just realized that we should keep the old format ("{}{}", without the underscore). Because it will change names for existing queries, which might result in existing visualizations breaking for users.